spec/collection-spec.coffee

```diff
@@ -1,18 +1,33 @@
-describe 'Brainstem.Collection', ->
-  collection = updateArray = null
+$ = require 'jquery'
+_ = require 'underscore'
+Backbone = require 'backbone'
```

Comment:
Since we made these global via the browserify-shim file, I believe we don't actually need to be explicit about this here. If we want to be explicit, then we should consider retracting the global exposure that we made in the shim declaration.